src/screens/SignUp.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import { NavLink, } from "react-router-dom";
+import { validateEmail, validateName, validatePassword } from "../assets/helpers/userValidationTools"
+
+
+class SignUp extends Component {
+   constructor(props) {
+      super(props);
+
+      this.state = {
+         firstName: '',
+         lastName: '',
+         email: '',
+         password: '',
+         passwordVerification: '',
+         formIsValid: true,
+         redirectToHome: false,
+
+      }
+
+      this.baseState = this.state
+      this.handleValidation = this.handleValidation.bind(this)
+      this.errorSpan = this.errorSpan.bind(this)
+
+   }
+
+   userSubmit(e) {
+      e.preventDefault();
+      console.log(this.state.formIsValid)
+      if (this.handleValidation()) {
+         let { firstName, lastName, email, password, } = this.state
+         console.log(this.state)
+         fetch('http://localhost:8080/users', {
```

Comment:
Since we have a little bit of time before the end of the sprint it would be good to start migrating out our API calls into a networking library

Reply:
I am not done with several stories, if you want to make a story for this I can do that instead, but I didnt get the impression it would be best to not complete the other items due this sprint.